### DISCUSSION TITLE
Time estimate for Snaps in stable MetaMask extension

### DISCUSSION THREAD

**Discussion by zwilling** [MetaMask/snaps#898] [Q&A] [answered]
Dear MetaMask team,

Is there a rough estimate when Snaps will be available to ordinary MetaMask users who do not have Flask installed?
How high is the chance that it will be available, for example, in Q1 2023 or 2023 in general?

I am asking because Snaps would be the perfect fit for a project about generating zero knowledge proofs for verifiable credentials. But to choose this solution, my team needs to know if Snaps are available to our users in the foreseeable future.

The resources I found just state that:
> [Snaps will eventually be enabled on both the stable extension and on MetaMask mobile.](https://docs.google.com/presentation/d/1LG8MqRrbb9qSg4m8ZjJXPQFccb9YPc-6387hSNpscpY/edit#slide=id.g1351d81a291_2_56)

> [Could I run a snap in my current MetaMask Extension? Not yet.](https://metamask.io/flask/)

Some [articles](https://cryptoslate.com/videos/metamask-snaps-could-change-the-face-of-web3-giving-dapps-access-to-btc-notifications-and-more/) claim that Snaps will be made available in the stable MetaMask extension later this year but without giving a source for this information.

I know that it is hard to give delivery estimates in software development and I see that a lot of [progress](https://github.com/MetaMask/SIPs) is being made. But it would really help projects like mine to utilize Snaps in our product design.

**Comment by Montoya** [comment, marked as answer]
Hi @zwilling, we are currently working on our roadmap for enabling Snaps in the stable extension. My best estimate is sometime in Q2 of 2023. Mobile will come some time after that. When I have a more specific estimate I will let you know.

**Reply by zwilling** [reply]
Thanks a lot @Montoya .
That is very helpful.
